src/backend/src/prisma/schema.prisma

```diff
@@ -97,10 +97,11 @@ model Session {
 }
 
 model User_Settings {
-  id           String @id @default(uuid())
-  userId       Int    @unique
-  user         User   @relation(fields: [userId], references: [userId])
-  defaultTheme Theme  @default(DARK)
+  id           String  @id @default(uuid())
+  userId       Int     @unique
+  user         User    @relation(fields: [userId], references: [userId])
+  defaultTheme Theme   @default(DARK)
+  slackId      String?
```

Comment:
we decided with the frontend ticket that instead of doing an optional slackId we're going to do a String with default empty string

to do this, delete your existing migration, change the schema.prisma, then run migrate again